apps/evm/lib/evm/refunds/sstore.ex

```diff
@@ -35,23 +39,35 @@ defmodule EVM.Refunds.Sstore do
       initial_value == current_value && initial_value == 0 ->
         0
 
-      initial_value == current_value && initial_value != 0 && new_value == 0 ->
+      initial_value == current_value && new_value == 0 ->
         15_000
 
-      initial_value != current_value && initial_value != 0 && current_value == 0 ->
-        -15_000
+      true ->
+        first_refund =
+          cond do
+            initial_value != 0 && current_value == 0 ->
+              -15_000
```

Comment:
can we pull these out into module variables like we did with the other gas values?

Reply:
done